web/services/createProjectsService.php

```diff
@@ -208,11 +209,33 @@
 
 
     if (count($createProjects) >= 1)
-        if (ProjectsFacade::CreateProjects($createProjects) == -1)
-            $string = "<return service='createProjects'><error id='1'>There was some error while creating the projects</error></return>";
+        if(count($createProjects) == 1){
+            $operationResult = ProjectsFacade::CreateProject($createProjects[0]);
+            if ($operationResult->isSuccessful == false){
```

Comment:
It's so weird to have to do this... Maybe because `isSuccessful` was never initialized in some cases? We could initialize it to false in the `OperationResult`.

Also, I'd use the getter and not the property directly.

Reply:
Sounds good to me. Will make the changes and test it out.